pcdsdevices/ccm.py

```diff
@@ -773,6 +773,41 @@ def inverse(self, real_pos: namedtuple) -> namedtuple:
         return self.PseudoPosition(energy=energy)
 
 
+class CCMEnergyWithACRStatus(CCMEnergyWithVernier):
+    """
+    CCM energy motor and ACR beam energy request with status.
+    Note that in this cae vernier indicates any ways that ACR will act on the
```

Comment:
```suggestion
    Note that in this case vernier indicates any ways that ACR will act on the
```

I think this is what you meant?  unless "cae" is yet another acronym I need to learn